process.md

```diff
@@ -4,53 +4,55 @@
 
 - [ ] Email them the questions
 - [ ] Get the response
-- [ ] Figure out a schedule
-- [ ] Put it in calendar
 
 ## Adding to Website
 
-- [ ] Create a post
+- [ ] Create a post by running:
+   
+   `rake post title="name" site="www.example.com" twitter="name" github="name" instagram="name"`
 
-    rake post title="name" site="" twitter="name" github="name" instagram="name"
+- [ ] Copy text into the post
+- [ ] Edit it, add `###` headers etc.
+- [ ] Edit metadata
 
 - [ ] Add photos
-  - [ ] Move them to src/img/posts/
+  - [ ] For PNG format, go to [tiny png](https://tinypng.com/) and shrink images 
+  - [ ] Place images in src/img/posts/
   - [ ] Name them name.jpg and name-alt.jpg (and name-alt2.jpg)
-  - [ ] Square the name.jpg one using Photoshop
+  - [ ] Square the name.jpg one using Photoshop. This will be used as the portrait
   - [ ] Resize to be modest
   - [ ] Run `gulp`
-  - [ ] Add image name to metadata in post
-  - [ ] Copy this into a nice part of the post:
-
-    <img src="{{ BASE_PATH }}/assets/img/posts/name-alt.jpg" class="inner-post-image" />
-
-  - [ ] Push photos up to the server, so that they'll be there in the test-run
-- [ ] Copy text into the post
-- [ ] Edit it, add `###` headers
-- [ ] Edit metadata
-- [ ] Test run. NB: Images don't work locally. See https://github.com/RichardLitt/nomad-as-fuck/issues/23
+  - [ ] Add image name.jpg to metadata in post
+  - [ ] Add images to a nice part of the post by copying this:
+  
+    <img data-src="{{ site.url }}/assets/img/posts/name-alt.jpg" class="inner-post-image lazyload" />
 
-    jekyll serve
+- [ ] Test run locally by running `jekyll serve`
+    - [ ] Review and edit 
 
+- [ ] Create a branch `feat/namePost` and push to GitHub
+- [ ] Log into netlify and launch deploy preview 
+    - [ ] Final review and edit (images will not display in preview deploy)
```

Comment:
Images should show now, shouldn't they?

Reply:
Images show on localhost and on nomadasfuck.com. Images for new posts will currently not show up in preview deploys. This is because mailchimp requires absolute paths for images to display. Jekyll site.url defaults to localhost:4000 when ran locally and nomadasfuck,com when ran in production.

Reply:
Can we have a process where preview deploys use a dev env?

Reply:
fixed in #66